docs/releases/npm-packages.md

```diff
@@ -0,0 +1,37 @@
+# Publishing @trezor package to npm registry
+
+`npm publish` could be done manually on [gitlab CI](https://gitlab.com/satoshilabs/trezor/trezor-suite/-/pipelines/) in `Deploy to dev` phase.
```

Comment:
Also, I'm thinking what if we create a new stage in the ci that would be `deploy to npm`?

Reply:
https://github.com/trezor/trezor-suite/issues/4946